packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -759,6 +768,16 @@ class SortedTable extends Component {
       <TableFilter filters={props.filters} onChange={this._setFilter} ref='filterInput' value={this._getFilter()} />
     )
 
+    const filterItemsPerPage = (
+      <DropdownButton bsStyle='info' id='itemsPerPage' title={itemsPerPage}>
```

Comment:
Why did you add the `id`?

Reply:
In the [react-bootstrap](https://react-bootstrap.netlify.app/components/dropdowns/#dropdown-button-props) documentation, its say the props id are `required` on the `DropDownButton` component

Reply:
Was it triggering an error or an issue with the button? It looks like they "require" it on many components for accessibility reasons. But anyway, it doesn't look related to this PR.

Reply:
it was triggering error in a web console.

Reply:
Ok then you can fix this (and check if there are other places with this isssue) but in another PR since it's not related to this one.